src/hotspot/cpu/aarch64/assembler_aarch64.hpp

```diff
@@ -3527,6 +3527,22 @@ void mvnw(Register Rd, Register Rm,
   INSN(sve_lastb, 0b1);
 #undef INSN
 
+// SVE cpy general-purpose register
+  void sve_cpy(FloatRegister Zd, SIMD_RegVariant T, PRegister Pg, Register Rn) {
+    starti;
+    assert(T != Q, "invalid size");
+    f(0b00000101, 31, 24), f(T, 23, 22), f(0b101000101, 21, 13);
+    pgrf(Pg, 10), srf(Rn, 5), rf(Zd, 0);
+  }
+
+// SVE INDEX (immediates)
+  void sve_index(FloatRegister Zd, SIMD_RegVariant T,
+            int16_t imm1, int16_t imm2) {
```

Comment:
int instead of int16_t?
Style: Merge into one line, or fix the second line "int16_t imm1..." indentation.

Reply:
OK. I will fix that.